### PR TITLE
#182282685 - DOCUMENT API USING SWAGGER

### PR DESCRIPTION
**Pull request by dalienst**
#### What does this Pull Request do?
Document the API using Swagger
#### Description of Task to be completed?
Documentation of the API
#### How should this be manually tested?
Run http://127.0.0.1:8000/swagger/ after starting the server
#### Any background context you want to provide?
The need to follow through what is happening in the backend
#### What are the relevant Pivotal Tracker?
[#182282685](https://www.pivotaltracker.com/n/projects/2570600/stories/182282685) 
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: speaksfer/urls.py [hunk 1]
```diff
@@ -15,7 +15,33 @@
...
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Speaksfer API",
+        default_version="v1",
+        description="Test description",
+        terms_of_service="https://www.speaksfer.com/policies/terms/",
+        contact=openapi.Contact(email="contact@speaksfer.local"),
+        license=openapi.License(name="Test License"),
```
[the remark this answers is not in the document]
Comment (dalienst) [reply speaksfer/urls.py:29]: What license should we use ama we create our own license

File: Pipfile [hunk 2]
```diff
@@ -11,6 +11,7 @@ django = "4.0.5"
 whitenoise = "6.2.0"
 gunicorn = "20.1.0"
 djangorestframework = "3.13.1"
+drf-yasg = "*"
```
[the remark this answers is not in the document]
Comment (dalienst) [reply Pipfile]: Done

File: speaksfer/urls.py [hunk 3]
```diff
@@ -15,7 +15,33 @@
...
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Speaksfer API",
+        default_version="v1",
+        description="Test description",
+        terms_of_service="https://www.speaksfer.com/policies/terms/",
```
Comment (mainamuriuki274) [inline speaksfer/urls.py:27]: add a valid url for the terms or remove

File: speaksfer/urls.py [hunk 4]
```diff
@@ -15,7 +15,33 @@
...
+        contact=openapi.Contact(email="contact@speaksfer.local"),
```
Comment (mainamuriuki274) [inline speaksfer/urls.py:28]: add valid email or emove

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: speaksfer/urls.py (hunk 3 above)
Comment (dalienst) [reply speaksfer/urls.py]: Done

File: speaksfer/urls.py (hunk 4 above)
Comment (dalienst) [reply speaksfer/urls.py]: Done

File: speaksfer/urls.py [hunk 5]
```diff
@@ -15,7 +15,31 @@
...
 from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Speaksfer API",
+        default_version="v1",
+        description="Test description",
```
Comment (mainamuriuki274) [inline speaksfer/urls.py:26]: Also hii usiite test give it a description

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: speaksfer/urls.py (hunk 5 above)
Comment (dalienst) [reply speaksfer/urls.py:26]: Sawa

**Pull request merged**